pkg/config/legacy/tests/config.py

```diff
@@ -191,7 +191,7 @@ def get_histogram_aggregates(configstr=None):
         for val in vals:
             val = val.strip()
             if val not in valid_values:
-                log.warning(f"Ignored histogram aggregate {val}, invalid")
+                log.warning("Ignored histogram aggregate %s, invalid", val)
```

Comment:
why removing the f string literals?

Reply:
Good question! We should avoid using f-strings in logger for performance reasons: https://docs.astral.sh/ruff/rules/logging-f-string/ 

But I agree here it's not a big deal though since this is a test

Reply:
Is this spotted by `ruff`? Amazing!

Reply:
Yes it is 🙂